src/main/scala/com/databricks/spark/csv/util/TypeCast.scala

```diff
@@ -35,7 +38,8 @@ object TypeCast {
    * @param datum string value
    * @param castType SparkSQL type
    */
-  private[csv] def castTo(datum: String, castType: DataType, nullable: Boolean = true): Any = {
+  private[csv] def castTo(datum: String, castType: DataType, nullable: Boolean = true,
```

Comment:
Unfortunately, MiMa checker fails on this although it is ```private[csv]```. Please Put a style checker exclusion clause around this line to fix the MiMa failure.